src/udev/net/link-config.c

```diff
@@ -1314,6 +1357,36 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (streq(rvalue, "all")) {
```

Comment:
Please also handle the case that an empty string is passed.

Reply:
Empty string will be handled by `parse_cpu_set_extend` which then call `parse_cpu_set_full`. I'm confused here.